docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -0,0 +1,872 @@
+
+
+metricbeat_outages_ecs::
+
+* For {metricbeat} data where `event.dataset` is `system.cpu` and 
+  `system.filesystem`.
+* Models counts of {metricbeat} documents 
+  (`partition_field_name` is `event.dataset`).
+* Detects unusual decreases in {metricbeat} documents.
+
+Influencers:
+
+* `event.dataset`
+
+Bucket span: 10m.
+
+Function: `low_count`.
+
+
+[float]
+[[ootb-ml-jobs-nginx]]
+=== Nginx
+
+These {anomaly-job} wizards appear in {kib} if you use {filebeat} to ship access 
+logs from your http://nginx.org/[Nginx] HTTP servers to {es} and store it using 
+fields and datatypes from the Elastic Common Schema (ECS).
+
+low_request_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `nginx.access`.
+* Models the event rate of http requests. 
+* Detects unusually low counts of HTTP requests compared to the previous event 
+rate.
+
+This job can be created if web access log data exists matching the defined 
+filter.
+
+Bucket span: 15m.
+
+Function: `low_count`.
+
+
+source_ip_request_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `nginx.access`.
+* Models the event rate of HTTP requests by source IP.
+* Detects source IPs with unusually high request rates in the HTTP access log 
+  compared to the previous rate. 
+
+Influencers:
+
+* `source.address`
+
+Bucket span: 1h.
+
+Function: `high_count`.
+
+
+source_ip_url_count_ecs::
+
+* For HTTP web access logs where `event.dataset` is `nginx.access`.
+* Models the event rate of HTTP requests by source IP.
+* Detects source IPs with unusually high distinct count of URLs in the HTTP 
+  access log.
+
+Influencers:
+
+* `source.address`
+
+Bucket span: 1h.
+
+Function: `high_distinct_count`.
+
+
+status_code_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `nginx.access`.
+* Models the occurrences of HTTP response status codes (`partition_field_name` 
+  is `http.response.status_code`).
+* Detects unusual status code rates in the HTTP access log compared to previous 
+  rates.
+
+Influencers:
+
+* `http.response.status_code` 
+* `source.address`
+
+Bucket span: 15m.
+
+Function: `count`.
+
+
+visitor_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `nginx.access`.
+* Models visitor rates.
+* Detects unusual visitor rates in the HTTP access log ompared to previous 
+  rates.
+
+Bucket span: 15m.
+
+Function: `non_zero_count`.
+
+
+[float]
+[[ootb-ml-jobs-siem]]
+=== SIEM
+
+linux_anomalous_network_activity_ecs::
+windows_anomalous_network_activity_ecs::
+
+* For network activity logs where `agent.type` is `auditbeat` or `winlogbeat`.
+* Models the occurrences of processes that cause network activity.
+* Detects network activity caused by processes that occur rarely compared to 
+  other processes.
+
+Looks for unusual processes using the network which could indicate
+command-and-control, lateral movement, persistence, or data exfiltration
+activity.
+
+This job can be created if auditbeat or winlogbeat data exists, matching the 
```

Comment:
I think this sentence is redundant after you add the intro to the SIEM section (since otherwise it just duplicates the first bullet):
```suggestion
This job can be created if auditbeat or winlogbeat data exists, matching the 
```